dmutils/content_loader.py

```diff
@@ -90,7 +217,10 @@ def __init__(self, manifest, content_directory):
         ]
 
     def get_question(self, question):
-        return self._questions.get(question, {}).copy()
+        q = self._questions.get(question, {}).copy()
+        if q:
+            q['id'] = question
```

Comment:
I think this would benefit from an explanation of what this bit is for.
All items in `_questions` come from `_load_questions` which adds the `'id'`, so does this have something to do with `serviceTypes*`?